commit 4a007f6c1a2f601a88262255c802e5b20edfd2a7
Author: Ivan Murashov <ivan.murashov@lge.com>
Date:   Fri May 9 11:54:34 2025 -0700

    IWYU: Add missing includes in span_forward_internal.h
    
    In the CL https://crrev.com/c/6418567 used std::numeric_limits and
    size_t in the file base/containers/span_forward_internal.h, but
    corresponding includes were missed.
    Added missing includes for std::numeric_limits and size_t usage.
    
    Bug: 41455655
    Change-Id: If9bd8dd3772ee707e220ff94bb322b776d0ecd3a
    Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/6513124
    Reviewed-by: Mark Mentovai <mark@chromium.org>
    Commit-Queue: Ivan Murashov <ivan.murashov@lge.com>
    Reviewed-by: Daniel Cheng <dcheng@chromium.org>
    Cr-Commit-Position: refs/heads/main@{#1458258}

diff --git a/base/containers/span_forward_internal.h b/base/containers/span_forward_internal.h
index 65cf58c927fe9..96ffc346d97f0 100644
--- a/base/containers/span_forward_internal.h
+++ b/base/containers/span_forward_internal.h
@@ -5,6 +5,10 @@
 #ifndef BASE_CONTAINERS_SPAN_FORWARD_INTERNAL_H_
 #define BASE_CONTAINERS_SPAN_FORWARD_INTERNAL_H_
 
+#include <stddef.h>
+
+#include <limits>
+
 namespace base {
 
 // [span.syn]: Constants
